modules/rules_python/metadata.json

```diff
@@ -21,9 +21,8 @@
         "0.12.0",
         "0.14.0",
         "0.15.0",
-        "0.15.1"
+        "0.15.1",
+        "0.16.1"
     ],
-    "yanked_versions": {
```

Comment:
@kormide Do you think it would be reasonable to make `yanked_versions` a preserved field in the Publish to BCR app so that users don't have to maintain these entries in their project's template?

@meteorcloudy We might need yet another verification check :-)

Reply:
Yes I think that's a good idea. I'll log an issue.

Reply:
https://github.com/bazel-contrib/publish-to-bcr/issues/35